synapse/logging/opentracing.py

```diff
@@ -85,14 +85,14 @@ def interesting_function(*args, **kwargs):
        return something_usual_and_useful
 
 
-Operation names can be explicitly set for functions by using
-``trace_using_operation_name``
+Operation names can be explicitly set for a function by passing the
+operation name to ``trace``
 
 .. code-block:: python
 
-   from synapse.logging.opentracing import trace_using_operation_name
+   from synapse.logging.opentracing import trace
 
-   @trace_using_operation_name("A *much* better operation name")
+   @trace(operation_name="A *much* better operation name")
```

Comment:
this isn't a great example, since the operation names we have use underscores, not long sentences.

Reply:
Sure